### DISCUSSION TITLE
Local fonts via custom css in custom typo

### DISCUSSION THREAD

**Discussion by deltapapa01** [templaza/astroid-framework#280] [Q&A] [answered]
Hello,
is it possible to add a css class to the custom typography setting in order to then call up this specific font via css class in the text?
Example: I call up a text with 
`<p class="myfont">TEXT</p> `
and then format this css class in the template manager in the custom typo.
Unfortunately, this does not work with the css selector.
At the moment I'm doing this via user.css and entering the custom fonts there.

**Comment by Chacapamac** [comment]
I @deltapapa01 
**I’m not to sure if the following can help You???**

Personally, I use a custom SCSS file for my custom CSS.
— SCSS give you the possibility to add your own custom variables and this is really useful.

— The following is variables that I use to pair a font to my css by a custom variable and others (just to show)
```
///* :::::::::::::::::::::TYPO VARIABLES */
///*Title & Text FONT FAMILY --------*/  
$title-font:"Source Sans Pro", Verdana, sans-serif;
$body-font:"Source Sans Pro", Verdana, sans-serif;
$compress-font:'Oswald', Verdana, sans-serif;

///* Text Weight*/ 
	$regular: 400;
	$semibold: 600;
	$bold: 700;
	$title-bold: 600;

///* Links & Menu Color --------*/
	$link: #fb6058; ///* Orange-Red */
	$link-hover: #1ebcd3; ///*Sky Blue */ 
	$link-menu: #35312e; ///* Warm Black */
	$link-menu-hover: #fb6058; ///* Orange-Red */
	$light-link: #f6fbfb; ///*Pale Blue */

///* Site Colors --------*/
	$mauve: #8548A9;
	$medium-mauve: #D1BBDF;
	$light-mauve: #E8DDEF;
	$fluo-green: #95cf1e;
	$light-green: #d0e9c6;
	$medium-green: #359895;
	$dark-green: #116360;

```
In all your custom SCSS/CSS you can simply call a Typo by using normal CSS classes like.
```
 ///* HEADER HEADING FONT - Family */
 h1, h2, h3, h4 { font-family: $title-font; letter-spacing:-.01em; line-height:1.2em;}
```

What is really powerful is that you can pair a color, a typography, font size, font weight and more to a simple, easy to remember placeholder (variable). After if you want to change the font in all your custom CSS, you just change your variable and ALL the custom SCSS custom element with that variable will be change.

I know CSS have a variable system too....

**Comment by sonvnn** [comment, marked as answer]
Hi @deltapapa01 

You can use Custom Typography. 
Go to Template Options >> Typography >> Custom Typography.
![Screen Shot 2023-01-19 at 11 18 55](https://user-images.githubusercontent.com/3204103/213354237-ff4aa1a3-89b9-450e-8981-37693250f8fe.png)

Hope this can help.

Best Regards,
Sonny

**Comment by deltapapa01** [comment]
Hi Sonny, Hi Chacapamac,
thank you both for the answer. 
I forgot the dot in the class, so the class didn't move. Now it works fine.
That with the SCSS is a great tip, I'll definitely take a closer look at it.

Thank you both,
deltapapa
